jobs/datadog-cc-worker/templates/process.yaml.erb

```diff
@@ -0,0 +1,6 @@
+init_config:
+
+instances:
+  - name: cc api worker
```

Comment:
This proposal `cc.jobs.local.number_of_workers` is for paas-cf, not this repo.

I fixed the rest.